ISLE/stb_image.h

```diff
@@ -0,0 +1,7988 @@
+/* stb_image - v2.30 - public domain image loader - http://nothings.org/stb
```

Comment:
This is a 3rd party dependency and should be located in the `3rdparty` folder.

I'd strongly encourage exploring alternatives to pulling in another dependency, i.e. using SDL, or using alternative methods. I'd rather not pull in a new dependency for a fringe feature such as this.

Reply:
SDL has build in support for loading BMP files, there is no need for extra code for this. All graphic in the game is also coming from BMP files so the game has custom code for it was well